docs/source/model_design/designing_vivarium_model/closed_versus_open_model_cohorts/index.rst

```diff
@@ -33,8 +33,92 @@ Closed Versus Open Vivarium Cohorts
 
 .. todo:: 
 
-  Choosing between open and closed cohorts
+  Figure out how to frame this page within a greater 'Designing Your Demography' section
 
-  How to interpret results differently
+Cohort analysis in microsimulation
+----------------------------------
 
-  How decision may impact results
\ No newline at end of file
+A cohort in a real-world study refers to a group of people who are observed over a period, often to examine
+how specific exposures, conditions, or treatments affect them over time. For example, a cohort might 
+consist of individuals born in the same year or individuals diagnosed with a specific 
+disease at the same time. 
+
+When Vivarium researchers use the phrase 'cohort analysis', it means something a little different than how 
+a real-world study might define it.  In the context of microsimulation, the term 'cohort' is used to describe 
+a group of simulated individuals who share certain baseline characteristics or are subject to the same set of 
+rules and conditions over the simulation period. Microsimulation models often track these cohorts to project 
+outcomes under various scenarios. 
+
+In microsimulation, we can think of two distinct types of cohort analysis: open and closed cohort analysis.
+These two types essentially differ based on how fertility is enacted in the demographics of our microsimulation. 
+
+Open cohort analysis 
```

Comment:
I've got an idea that open-cohort analysis and closed-cohort analysis should be hyphenated, but I'm not going to make all the suggestions on it, because I'm doubting myself now.

Reply:
```suggestion
Open-cohort analysis 
```